packages/core/stories/button.doc.stories.mdx

```diff
@@ -30,6 +30,8 @@ A Button executes an action when clicked. Button labels should use only one or t
 - **Primary** is the most commonly-used button.
 - **Secondary** performs an action less likely to be triggered.
 
+If the use case requires the user to navigate to a new page or window rather than trigger a function, use a `Link` component instead. The exception to this behaviour is if the navigation is the core action on the page. In this case, a `CTA Button` should be used to trigger the navigation.
```

Comment:
```suggestion

##### Use when
- You want to indicate actions that the user can take.
- You need to highlight navigation as the core action on the page. In this case, a `CTA Button` should be used to trigger the navigation.

##### Don't use when

The use case requires the user to navigate to a new page or window rather than trigger a function, use a `Link` component instead.
```

Reply:
@sion-docs @joshwooding does that look better?

Reply:
Looks good 👍🏽 I would double check "You need to highlight navigation as the core action on the page. In this case, a `CTA Button` should be used to trigger the navigation." with @maj-stella

Reply:
That's great @Fercas123

Reply:
@joshwooding I updated it to the previous wording that @maj-stella  had already reviewed and approved.